### ISSUE TITLE
Bold Text Formatting Inherited by Bulleted List Level in WYSIWYG Editor

### ISSUE THREAD

**Issue by bussardrobbie** [BookStackApp/BookStack#4659]
### Describe the Bug

Good morning BookStack team,
I first wish to thank all contributing members for their time and dedication in maintaining the BookStack software. The software is fantastic and is very much appreciated!

I wish to report an issue where Paragraph-level Bold text formatting is inherited in the WYSIWYG editor between list levels and is unable to be changed in lower levels unless cleared in higher levels before descending. This issue seems to be limited to the Bold text formatting type (it does not happen with italics or underlining) and occurs regardless of whether UI buttons or hotkeys are used, lending to my belief that this issue is not related to the buttons or the hotkeys, but instead the formatting is not able to be cleared from being inherited from higher levels in the list.

Please see attached video for visual description / example.

### Steps to Reproduce

1. Create a new Article.
2. Utilize the WYSIWYG editor to create a Bulleted List.
3. Press Ctrl+B or press the Bold button to bold the text in the current level.
4. Press "Enter" and then "Tab" to begin a new list sublevel.
5. Type something on the new sublevel - observe that the text has inherited the Bold formatting.
6. Attempt to clear the Bold formatting from the text on the new level using the Ctrl+B hotkey or the Bold formatting toggle at the top bar.

### Expected Behaviour

The Bold text formatting should either:
1) Be reset on a list sublevel (not optimal as user may expect bold formatting to carry over)
2) Be able to be toggled off independent of the formatting of higher level text in the bulleted list.

### Screenshots or Additional Context

https://github.com/BookStackApp/BookStack/assets/65207496/83c171da-0b70-45ff-9afa-ed794ecfa540

### Browser Details

Microsoft Edge Business, Version 119.0.2151.44 (Official build) (64-bit)

### Exact BookStack Version

v23.06.2

**Comment by ssddanbrown** [comment]
Thanks for reporting @bussardrobbie,
Could confirm this. I found the behaviour to be somewhat intermittent depending on how the initial boldening selection was performed, and even things like the use of undo/redo could make this occur.
Seems to be due to styles being applied to the list item itself.

I found an upstream verified issue for this bug here:
https://github.com/tinymce/tinymce/issues/8313
